### DISCUSSION TITLE
Copilot returning 0/10 solutions

### DISCUSSION THREAD

**Discussion by adamerose** [community/community#47307] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I am finding that very often I get no suggestions from Copilot in contexts with pretty straightforward code, like below in line 15
<img width="1278" alt="image" src="https://user-images.githubusercontent.com/10884874/218631140-3d1cbd78-946b-4fd4-b941-5b8abcee738a.png">

Logs
<details>
[INFO] [solutions] [2023-02-14T03:23:21.638Z] prompt: {"prefix":"// Path: src\\components\\App.tsx\nimport { useState } from 'react';\nimport './App.css';\n\nfunction App() {\n  const [count, setCount] = useState(0);\n\n  const pages = [\n    { name: 'Home', path: '/' },\n    { name: 'About', path: '/about' },\n  ];\n  return (\n    <nav>\n      {/* react router nav bar */}\n      {pages.map((page) => (\n","suffix":"))}\n    </nav>\n  );\n}\n\nexport default App;\n","isFimEnabled":true,"promptElementRanges":[{"kind":"PathMarker","start":0,"end":32},{"kind":"BeforeCursor","start":32,"end":334}]}
[INFO] [default] [2023-02-14T03:23:21.639Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2023-02-14T03:23:21.835Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 196 ms
[INFO] [streamChoices] [2023-02-14T03:23:21.986Z] request done: headerRequestId: [3d71692f-a5ec-4503-9883-982979384e87] model deployment ID: [x625ba4f78aa8]

</details>

Diagnostics
<details>
## Configuration

- HTTP proxy: 
- HTTP proxy autentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

## Environment

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Feature Flags

- Self Signed Certificates: disabled
- Send Restricted Telemetry: enabled

## Extensions

- Copilot Version: 1.73.8685
- Is `win-ca` installed?: true
- Is `mac-ca` installed?: true

## Node setup

- Number of root certificates: 131
- Operating system: Windows_NT
- Operating system version: 10.0.22621
- Operating system architecture: x64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Reachability

- github.com: HTTP 200 - 
- copilot-proxy.githubusercontent.com: HTTP 200 - 
- default.exp-tas.com: HTTP 200 - 
</details>

**Comment by davecheney** [comment]
@adamerose thank you for the excellent bug report.

**Reply by davecheney** [reply]
@adamerose I have escalated this internally
